packages/datadog-plugin-rhea/test/index.spec.js

```diff
@@ -0,0 +1,544 @@
+
+              client.on('message', msg => {
+                msg.delivery.accept()
+                p.then(done, done)
+              })
+              serverContext.sender.send({ body: 'hello' })
+            })
+
+            it('receiving accepting', done => {
+              const p = expectReceiving(agent)
+
+              client.on('message', msg => {
+                process.nextTick(() => {
+                  msg.delivery.accept()
+                  p.then(done, done)
+                })
+              })
+              serverContext.sender.send({ body: 'hello' })
+            })
+
+            it('receiving rejecting', done => {
+              const p = expectReceiving(agent, 'rejected')
+
+              client.on('message', msg => {
+                process.nextTick(() => {
+                  msg.delivery.reject()
+                  p.then(done, done)
+                })
+              })
+              serverContext.sender.send({ body: 'hello' })
+            })
+
+            it('receiving releasing', done => {
+              const p = expectReceiving(agent, 'released')
+
+              client.on('message', msg => {
+                process.nextTick(() => {
+                  msg.delivery.release()
+                  p.then(done, done)
+                })
+              })
+              serverContext.sender.send({ body: 'hello' })
+            })
+
+            it('receiving modified', done => {
+              const p = expectReceiving(agent, 'modified')
+
+              client.on('message', msg => {
+                process.nextTick(() => {
+                  msg.delivery.modified()
+                  p.then(done, done)
+                })
+              })
+              serverContext.sender.send({ body: 'hello' })
+            })
+          })
+        })
+
+        describe('disconnect receiver side', () => {
+          beforeEach(() => agent.load(plugin, 'rhea'))
+
+          beforeEach(done => {
+            const rhea = require(`../../../versions/rhea@${version}`).get()
+
+            server = rhea.create_container()
+            client = rhea.create_container()
+
+            let sendables = 0
+
+            server.once('sendable', _context => {
+              serverContext = _context
+              if (++sendables === 2) done()
+            })
+            client.once('sendable', _context => {
+              clientContext = _context
+              if (++sendables === 2) done()
+            })
+            const listener = server.listen({ port: 0 })
+            listener.on('listening', () => {
+              connection = client.connect(Object.assign({ reconnect: false }, listener.address()))
+              connection.open_receiver({ autoaccept: false })
+              connection.open_sender()
+            })
+          })
+
+          it('sender span gets closed', (done) => {
+            const err = new Error('fake protocol error')
+            err.stack = 'a short, fake stack' // to not pollute test logs
```

Comment:
Is there a way to not print it at all instead?

Reply:
`rhea` *always* logs when there's an error thrown in this case (and a few others). I'll see if I can turn logging off.